From 54270a91feadc2f6156548f65bd4b96f3bc6eebd Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:13:05 -0400
Subject: [powerpc] update comments for generic idle conversion

Message-id: <1410545655-205645-557-git-send-email-dzickus@redhat.com>
Patchwork-id: 94516
O-Subject: [RHEL7 PATCH 556/626] powerpc: update comments for generic idle conversion
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 0d2b7ea9287d39e87531d233ba885263e6160127
Author: Geert Uytterhoeven <geert+renesas@glider.be>
Date:   Fri Jun 6 14:38:33 2014 -0700

    powerpc: update comments for generic idle conversion

    As of commit 799fef06123f ("powerpc: Use generic idle loop"), this
    applies to arch_cpu_idle() instead of cpu_idle().

    Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
    Cc: Benjamin Herrenschmidt <benh@kernel.crashing.org>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/irq.c b/arch/powerpc/kernel/irq.c
index 464a4a3..c1fdaa4 100644
--- a/arch/powerpc/kernel/irq.c
+++ b/arch/powerpc/kernel/irq.c
@@ -304,7 +304,7 @@ void notrace restore_interrupts(void)
  * being re-enabled and generally sanitized the lazy irq state,
  * and in the latter case it will leave with interrupts hard
  * disabled and marked as such, so the local_irq_enable() call
- * in cpu_idle() will properly re-enable everything.
+ * in arch_cpu_idle() will properly re-enable everything.
  */
 bool prep_irq_for_idle(void)
 {
-- 
1.7.1